### DISCUSSION TITLE
Lazygit Markdown bug

### DISCUSSION THREAD

**Discussion by Xyndra** [community/community#102201] [Mobile]
The Lazygit Readme looks very buggy for me

**Comment by stevepopovich** [comment]
Hi @Xyndra are you seeing this in the GitHub mobile app? Are you on iOS or Android? Do you have a link to the repository you are referring to?

**Reply by Xyndra** [reply]
https://github.com/jesseduffield/lazygit on Android

**Reply by stevepopovich** [reply]
Ok, thanks. On my device, things look ok. Can you describe, screenshot or video what you are seeing?

**Comment by MatyaGlory** [comment]
Hi
